includes/image/BaseImage.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-class WPNotify_BaseImage implements WPNotify_Image, WPNotify_JsonUnserializable {
```

Comment:
I didn't see the `WPNotify_JsonUnserializable` interface use here, sorry. Anyway, it is better how it is declared right now.